challenges/2-palindromos/Java/backtoaldo/palindromos.java

```diff
@@ -0,0 +1,29 @@
+import java.util.*;
+
+public class palindromos {
```

Comment:
```suggestion
public class Palindromos {
```

Um ponto que não é necessário mas fica como sugestão e boa prática é a classe ter o nome maiúsculo mas isso não é critério para validação! Então se quiser deixar assim não tem problema ok?